server/channels/store/sqlstore/outgoing_oauth_connection_store.go

```diff
@@ -82,6 +83,10 @@ func (s *SqlOutgoingOAuthConnectionStore) GetConnections(c request.CTX, filters
 		query = query.Where("Id > ?", filters.OffsetId)
 	}
 
+	if filters.Audience != "" {
+		query = query.Where("Audiences LIKE ?", fmt.Sprint("%", filters.Audience, "%"))
```

Comment:
Could we use `sq.Like` here?

Reply:
Should I use `sq.X` in other `Where` clauses as well?

Reply:
Ideally yes but up to you since that would be slightly out of scope for the PR.